### PR TITLE
Fixes #772 Toggling darkness off prevents ghosts from seeing one another

### LINKED ISSUE #772: [Ghost] Toggling darkness off makes other ghosts invisible
##### Revision:
Unknown

##### Server:
Sibyl 1 (Approx. 15:45 14 June UTC+12)

###### What did you expect to happen? 
Toggling darkness wouldn't make other ghosts invisible

###### What happened instead?
The darkness disappeared, but so did other ghosts

###### Why is this bad/What are the consequences?
Sometimes it's interesting being able to see how many other people are spectating the same people as you are

##### Steps to reproduce the problem:
1. Ghost
2. Find another ghost
3. Toggle darkness

Result: The other ghost should disappear

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Fixes #772 

Abuses client.images to get around the issue with flat see_invisible scaling not allowing for both being able to see ghosts while not seeing darkness layers.

Also adds a new toggle ghost visibility verb for users who want the old functionality.

### CODE REVIEW

**Comment by Chocobro** [comment]
wow that's an old bug

**Comment by Mandurrrh** [comment]
ayyyy :+1:

**Comment by ike709** [comment]
finally fixed this

File: code/modules/mob/dead/observer/logout.dm [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /mob/dead/observer/Logout()
+	client.images -= ghostimages
```
Comment (d3athrow) [inline code/modules/mob/dead/observer/logout.dm]: If the person is logging out the client goes away anyways, so for what porpoise?

Comment (RemieRichards) [inline code/modules/mob/dead/observer/logout.dm]: shutup smartypants

Comment (d3athrow) [inline code/modules/mob/dead/observer/logout.dm]: Bawk

**Comment by TZK13** [comment]
Wait please no I use this "bug" to able to see things without having ghosts in the way, it's useful as fuck for taking screenshots and shit.

File: code/modules/mob/dead/observer/logout.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/mob/dead/observer/logout.dm]: *throws chicken feed*

Comment (MrStonedOne) [inline code/modules/mob/dead/observer/logout.dm]: logout is for mob logouts, not client disconnects.

**Comment by MrStonedOne** [comment]
@TZK13 i added a verb to do that before you edited.

Making it so you don't see yourself is a touch harder, I could but it wouldn't fit in to the see ghosts verb, and i don't think its worth adding yet another verb. this at least restored old functionality for those who want it.

**Comment by Anonus** [comment]
How? You're a wizard oh my god you're a wizard

**Comment by TZK13** [comment]
Thank you for adding see ghosts as an option, I agree that while it would be nice there's not really a need for another verb just to hide yourself.

**Comment by xxalpha** [comment]
Worst bug. :+1: x :100:

**Comment by Jordie0608** [comment]
Mso chaaaaaangelog

**Comment by Jordie0608** [comment]
Changelog was incorporated into #8090

**Comment by optimumtact** [comment]
Awful fix

**Pull request merged**